v2.0/drop-sequence.md

```diff
@@ -28,12 +28,12 @@ table td:first-child {
 -----------|------------
 `IF EXISTS` |  Drop the sequence only if it exists; if it does not exist, do not return an error.
 `sequence_name` | The name of the sequence you want to drop. Find the sequence name with `SHOW CREATE TABLE` on the table that uses the sequence.
-`CASCADE` / `RESTRICT` | Not yet implemented. Currently, you can drop a sequence even if it has a dependency.
+`RESTRICT` | _(Default)_ Do not drop the sequence if any objects (such as [constraints](constraints.html) and tables) depend on it.
+`CASCADE` | Not yet implemented. Currently, you can only drop a sequence if it has no dependencies.
```

Comment:
Should be `Currently, you can only drop a sequence if nothing depends on it`. Sequences never have dependencies (i.e. they don't depend on anything).

Reply:
Done